### DISCUSSION TITLE
Old torrents got redownloaded after docker container is newly created

### DISCUSSION THREAD

**Discussion by alneven** [haugene/docker-transmission-openvpn#1954] [Q&A] [answered]
hello there,
I have this strange new behavior if the docker image got updated (the last 2 times) all **my old and REMOVED** torrents got automatically added/resumed and **downloaded AGAIN** to the default /downloads folder

It's a problem as it destroys my ratio ;) as the first time I didnt checked the client for days and it downloaded over 1TB stuff again.
Today morning it was the same issue at 6 am, until I could stop all the downloads at 10:30 am.

Is there any config to prevent this?
Is this a known issue?
Is there anyone else with the same issue?

I'm using this docker container since 2018 and watchtower automatically updates it and starts it.

In the log I can only see that 
Transmission 2.94 starts
It starts to using the settings from /data/transmission-home
Configs the watch dir (which is empty)
and start to add torrent --> queued for verification
Loads the torrents (today it loaded 218 torrents) and starts to verify them, and as they didn't exists, starts to download them
to the default download
(as I'm using sonarr to add the torrents, it's different for Movies and TV, there even though the data exisits, it starts to check them only in the main folder, not in the Movies and TV folders, therefore nothing is there)

Thanks for checking.
Alneven

**Comment by pkishino** [comment]
See #1937

**Comment by alneven** [comment]
@pkishino I have the issue the other way around...

I have already deleted those torrents and the files a long time ago. But with the restart of the container and transmission they somehow got resumed. I don't need them, and I don't want to seed them. And they were re-downloaded completely as they wasn't on DISC.
This happened the last time and today as well.

Now I made the /downloads folder **unwritable** as a workaround.
And again I have removed all the torrent files from the transmission-home/resume and torrents folders

**Reply by alneven** [reply]
but maybe this is also because of the version change of transmission within the container, I will check the log later if I had already once the V3.0

**Comment by pkishino** [comment]
You most likely used the dev branch before which used 3.0, then the new
official release uses 2.95..
The diff in hashing causes this to occur. What you need to do is clear out
the transmission resume/cache parts ..

**Comment by pkishino** [comment, marked as answer]
Not just the torrents files but the resume files and cache

**Reply by alneven** [reply]
yeah, but somehow I missed a few
good that my workaround worked ;)

![image](https://user-images.githubusercontent.com/16620485/133779007-82270a25-e82f-4128-8cad-1296326952ae.png)
